pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[tool.poetry]
+name = "cookiecutter-python-template"
+version = "0.1.0"
+description = """Cookiecutter template for a python projects."""
+authors = ["Parth Barhanpurkar <parth.barhanpurkar@analog.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.10,<3.12"
+cookiecutter = "^2.2.0"
+flake8 = "^6.0.0"
+
+[tool.poetry.group.dev.dependencies]
+black = { version = "^23.1.0", allow-prereleases = true }
+pytest = "^8.0.0"
+pytest-cov = "^4.0.0"
+pytest-cookies = "^0.6.1"
+pytest-mock = "^3.3.1"
+pytest-xdist = "^2.1.0"
+typer-cli = "^0.0.13"
+mdformat = "^0.7.16"
+invoke = "^2.2.0"
+toml = "^0.10.0"
+types-toml = "^0.10.0"
+pre-commit = "^3.6.0"
+isort = "^5.12.0"
+ruff = "^0.2.0"
+
+[build-system]
+requires = ["poetry_core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.pytest.ini_options]
+testpaths = "tests"
+python_files = 'tests/*.py'
+
+# public pypi packgaes.
+[[tool.poetry.source]]
+name = "PyPI"
+priority = "primary"
+
+[[tool.poetry.source]]
```

Comment:
Let's remove this Artifactory connection from this file. We don't (and won't) need it. We can keep it in the `pyproject.toml` within the template subdirectory though.

Reply:
Removed